src/Modal/Modal.test.jsx

```diff
@@ -128,6 +129,26 @@ describe('<Modal />', () => {
       expect(wrapper.find('button')).toHaveLength(1);
     });
 
+    it('renders custom close button string', () => {
+      wrapper = mount(<Modal {...defaultProps} closeText={closeText} />);
+      const modalFooter = wrapper.find('.modal-footer');
+      const closeButton = modalFooter.find('button');
+
+      expect(closeButton).toHaveLength(1);
+      expect(closeButton.text()).toEqual(closeText);
```

Comment:
These two assertions are actually `true` in both cases right? 

I think the check here is that whatever the wrapping element around `closeText` is should only contain a string and **not** an element.

Reply:
My point wasn't that the check isn't useful - it is. My point was more that the assertions were not fully capturing the test case that was being described (which are now 👍 ) in that the assertions would've been valid for a `string` prop or an `element` prop.

I should've made it clear that I meant that the test was **missing** a test case.

Regardless, thanks for adding the extra check! 👏